hydroflow/src/scheduled/subgraph.rs

```diff
@@ -1,17 +1,17 @@
-use crate::scheduled::HandoffData;
+use crate::scheduled::Context;
 
 /**
  * Represents a compiled subgraph. Used internally by [Dataflow] to erase the input/output [Handoff] types.
  */
 pub(crate) trait Subgraph {
     // TODO: pass in some scheduling info?
-    fn run(&mut self, handoffs: &[HandoffData]);
+    fn run(&mut self, context: Context<'_>);
```

Comment:
I think this is fine, but I have a (possibly irrational) dislike for the term `Context` because it's so broad (I know I called them `SendCtx` and `RecvCtx` but I don't love those either). No super strong feels but I would prefer something more evocative if we could come up with it, like `DataHandles` or something. But probably better than that.